src/Evolution/Executables/GeneralizedHarmonic/EvolveGhSingleBlackHole.hpp

```diff
@@ -239,6 +269,8 @@ struct EvolutionMetavars : public GeneralizedHarmonicTemplateBase<3> {
                          Actions::AdvanceTime,
                          PhaseControl::Actions::ExecutePhaseChange>>>>>;
 
+  using dg_element_array = gh_dg_element_array;
+
```

Comment:
Why is this necessary?

Reply:
I like that 👍 Or pass it down as a template param.